README.md

```diff
@@ -12,6 +12,16 @@ If something doesn't work, then feel free to [submit an issue](https://github.co
 
 More reading about protocol handlers: http://pla.nette.org/en/how-open-files-in-ide-from-debugger.
 
+Installing on Linux
+===================
+
+> If you use Arch Linux (Arch, Manjaro, Antergos, etc.), you can install [phpstorm-url-handler](https://aur.archlinux.org/packages/phpstorm-url-handler/) package.
+
+1. clone repository [https://github.com/sanduhrs/phpstorm-url-handler](https://github.com/sanduhrs/phpstorm-url-handler)
+2. go to cloned folder
+3. read installation instructions in README.md
```

Comment:
Replace `README.md` with link to `README.md` of that other repo.

---

Without the link it looks like you're referring back to same README.md that you're reading right now.